DESCRIPTION

```diff
@@ -13,6 +13,8 @@ Description: Use this package to install all dependencies and
 License: MIT + file LICENSE
 URL: https://github.com/mlr-org/mlr3book
 BugReports: https://github.com/mlr-org/mlr3book/issues
+Depends:
+    R (>= 2.10)
```

Comment:
Why?

Reply:
> W  checking data for ASCII and uncompressed saves (349ms)
     Warning: package needs dependence on R (>= 2.10)

I guess saving the data in a compressed format requires >= 2.10? 
I can try to store data differently if you want.

Reply:
Format is fine. I was just wondering why you are depending on an ancient R version. I'd set this to R (>= 3.1.0) (same dependency as mlr3).

Reply:
Done